.gitignore

```diff
@@ -1,10 +1,48 @@
 # Cargo build directory
-/target
+target/
+debug/
+Carge.lock
```

Comment:
I think we should move this one to the `cli` folder (plus there is a typo, you typed `Carge` instead of `Cargo` :smile:). If I understand well, the lock file should be present in lib folders but absent from binary folders, see https://doc.rust-lang.org/cargo/guide/cargo-toml-vs-cargo-lock.html.

Reply:
> If I understand well, the lock file should be present in lib folders but absent from binary folders, 

so I should move the whole `.gitignore` to the `cli` folder?

Reply:
Mmmh no I think we should let it at the root folder, but we should just add a specific one for `Cargo.lock` in the `cli` folder.

Reply:
should be done now